blocks/library/block/edit-panel/index.js

```diff
@@ -10,66 +11,97 @@ import { keycodes } from '@wordpress/utils';
  */
 import './style.scss';
 
+/**
+ * Module constants
+ */
 const { ESCAPE } = keycodes;
 
-function ReusableBlockEditPanel( props ) {
-	const { isEditing, title, isSaving, onEdit, onChangeTitle, onSave, onCancel } = props;
+class ReusableBlockEditPanel extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.bindTitleRef = this.bindTitleRef.bind( this );
+		this.handleFormSubmit = this.handleFormSubmit.bind( this );
+		this.handleTitleChange = this.handleTitleChange.bind( this );
+		this.handleTitleKeyDown = this.handleTitleKeyDown.bind( this );
+	}
+
+	bindTitleRef( ref ) {
+		this.titleRef = ref;
+	}
+
+	componentDidMount() {
```

Comment:
Conventionally like to see all component lifecycle following the constructor, with no functions in between.

Reply:
👌 I've moved it in c7e444f64c3df8edc56ae9b309ee98902e3acc3c.